### PR TITLE
Packaging python-etcd

### PR DESCRIPTION
**Pull request by TimothyAsir**
Added Makefile and spec file to create python-etcd package

Signed-off-by: Timothy Asir J <tjeyasin@redhat.com>

### CODE REVIEW

**Comment by nthomas-redhat** [comment]
@r0h4n , can you please review and merge this

**Comment by r0h4n** [comment]
python-etcd is a global dependency, please put this PR on tendrl/commons

File: dependencies/Makefile [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+all: srpm
+
+clean:
+	rm -rf dist/
+	rm -rf $(NAME)-$(VERSION).tar.gz
+	rm -rf $(NAME)-$(VERSION)-1.src.rpm
+
+dist:
+	git clone https://github.com/jplana/python-etcd python-etcd-0.4.3
+	tar -zcvf python-etcd-0.4.3.tar.gz python-etcd-0.4.3
```
Comment (r0h4n) [inline dependencies/Makefile]: use $NAME and $VERSION instead of hard code

File: dependencies/Makefile [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+	rm -fr python-etcd-0.4.3
```
Comment (r0h4n) [inline dependencies/Makefile]: same: remove hardcoding

File: dependencies/Makefile (hunk 1 above)
Comment (TimothyAsir) [inline dependencies/Makefile]: Done

File: dependencies/Makefile (hunk 2 above)
Comment (TimothyAsir) [inline dependencies/Makefile]: Done

**Pull request closed without merging**